x-pack/metricbeat/module/oracle/tablespace/_meta/fields.yml

```diff
@@ -0,0 +1,54 @@
+- name: tablespace
+  type: group
+  description: tablespace
+  fields:
+    - name: name
+      type: keyword
+      description: Tablespace name
+
+    - name: data_file
+      type: group
+      description: Database files information
+      fields:
+        - name: id
+          type: long
+          description: Tablespace unique identifier
+        - name: name
+          type: keyword
+          description: Filename of the data file
+        - name: size
+          type: group
+          description: Size information about the file
+          fields:
+            - name: max.bytes
```

Comment:
The fields that are in bytes will be nice to have `format: bytes`

Reply:
Ohhhh!! So true... Changing

Reply:
No, wait. Is there a `bytes` datatype or a type of the size of a byte? https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-types.html